test/promises_aplus_adapter.js

```diff
@@ -13,11 +14,9 @@ require( "jsdom" ).env( "", function ( errors, window ) {
 		var deferred = jQuery.Deferred();
 
 		return {
-			get promise() {
-				return deferred.promise();
-			},
+			promise: deferred.promise(),
```

Comment:
Is this acceptable? I thought something about promises-aplus-tests might have required a getter here.